### PR TITLE
Add outdoors sport link to unit highlights

### PR DESCRIPTION
**Pull request by mikkoisa**
Missing translations

### CODE REVIEW

**Comment by mikkoisa** [comment]
[(2) Palvelukartan ja Ulkoliikuntapaikan välinen linkki](https://trello.com/c/6SuQtSlK/781-2-palvelukartan-ja-ulkoliikuntapaikan-v%C3%A4linen-linkki)

**Comment by mikkoisa** [comment]
[(2) Palvelukartan ja Ulkoliikuntapaikan välinen linkki](https://trello.com/c/6SuQtSlK/781-2-palvelukartan-ja-ulkoliikuntapaikan-v%C3%A4linen-linkki)

File: config/default.js [hunk 1]
```diff
@@ -170,6 +170,7 @@ export default {
   "ortographicMapURL": settings.ORTOGRAPHIC_MAP_URL,
   "guideMapURL": settings.GUIDE_MAP_URL,
   "reittiopasURL": settings.REITTIOPAS_URL,
+  "ulkoliikuntaURL": settings.ULKOLIIKUNTA_URL,
```
Comment (Dremars) [inline config/default.js:173]: I think we could change these to english to keep consistency

File: .env.example [hunk 2]
```diff
@@ -18,6 +18,7 @@ SERVICE_MAP_URL="https://tiles.hel.ninja/styles/hel-osm-bright/{z}/{x}/{y}"
 ACCESSIBLE_MAP_URL="https://tiles.hel.ninja/styles/turku-osm-high-contrast-pattern/{z}/{x}/{y}"
 ORTOGRAPHIC_MAP_URL="https://kartta.hsy.fi/geoserver/gwc/service/wmts?layer=taustakartat_ja_aluejaot:Ortoilmakuva_2017&tilematrixset=ETRS-GK25&Service=WMTS&Request=GetTile&Version=1.0.0&TileMatrix=ETRS-GK25:{z}&TileCol={x}&TileRow={y}&Format=image/jpeg"
 GUIDE_MAP_URL="https://kartta.hel.fi/ws/geoserver/avoindata/gwc/service/wmts?layer=avoindata:Karttasarja_PKS&tilematrixset=ETRS-GK25&Service=WMTS&Request=GetTile&Version=1.0.0&TileMatrix=ETRS-GK25:{z}&TileCol={x}&TileRow={y}&Format=image%2Fpng"
+ULKOLIIKUNTA_URL="https://ulkoliikunta.fi"
```
Comment (Dremars) [inline .env.example:21]: I think we could change these to english to keep consistency

File: server/server.js [hunk 3]
```diff
@@ -185,6 +185,7 @@ const htmlTemplate = (reactDom, preloadedState, css, cssString, locale, helmet,
         window.nodeEnvSettings.ORTOGRAPHIC_MAP_URL = "${process.env.ORTOGRAPHIC_MAP_URL}";
         window.nodeEnvSettings.GUIDE_MAP_URL = "${process.env.GUIDE_MAP_URL}";
         window.nodeEnvSettings.REITTIOPAS_URL = "${process.env.REITTIOPAS_URL}";
+        window.nodeEnvSettings.ULKOLIIKUNTA_URL = "${process.env.ULKOLIIKUNTA_URL}";
```
Comment (Dremars) [inline server/server.js:188]: I think we could change these to english to keep consistency

File: src/views/UnitView/components/Highlights/Highlights.js [hunk 4]
```diff
@@ -2,15 +2,40 @@ import React from 'react';
...
   }
 
+  if (showOutdoorsLink) {
+    const outdoorsObject = {
+      id: 'outdoorSports',
+      value: {
+        www: {
+          fi: `${config.ulkoliikuntaURL}/unit/${unit.id}`,
+          en: `${config.ulkoliikuntaURL}/unit/${unit.id}`,
+          sv: `${config.ulkoliikuntaURL}/unit/${unit.id}`,
```
Comment (Dremars) [inline src/views/UnitView/components/Highlights/Highlights.js:27]: I think we could change config.ulkoliikuntaURL to english to keep consistency

**Review by Dremars — CHANGES_REQUESTED** [review]

**Review by Dremars — APPROVED** [review]

**Pull request merged**